src/SharedElementRenderer.js

```diff
@@ -56,7 +56,7 @@ class SharedElementRenderer extends PureComponent {
 
     const animations = [];
 
-    if (source.position.pageY !== destination.position.pageY) {
+    if (source.position.pageY && destination.position.pageY) {
```

Comment:
I think the intention of the comparison is to also avoid an animation and render if the `pageY` values are identical. And also, it should be able to handle the `0` case, which is falsy and does not pass your new test. So perhaps change it to this which takes care of your issue as well as satisfies the original intent:

```
if (source.position.pageY !== undefined
    && destination.position.pageY !== undefined
    && source.position.pageY !== destination.position.pageY)
{
```

Reply:
Or even more economically (without such a long IF condition) as in the updated merge request.

Reply:
You were right, the re-render was needless. The pull request was modified accordingly.